tools/get_release_contributors.py

```diff
@@ -89,36 +96,36 @@ def get_login_or_realname(self, key, commit_hash):
         login = self._get_login(key, commit_hash)
         if login:
             return f'@{login}'
-        m = re.match('\A([^@<>]+) <.*>\Z', key)
-        if m:
-            return m.group(1)
-        logger.warning(f'unable to extract github login or real name from {repr(key)}')
+        matches = re.match(r'\A([^@<>]+) <.*>\Z', key)
+        if matches:
+            return matches.group(1)
+        logger.warning("unable to extract GitHub login or real name from %r", key)
         return None
 
-    def get_user_by_commit(self, hash):
+    def get_user_by_commit(self, sha):
         """
-        Retrieves the associated Github user name for the given commit
-        hash.
+        Retrieves the associated GitHub username for the given commit hash.
         """
-        r = self._github_api_get(f'repos/jamulussoftware/{self.repo}/commits/{hash}')
-        if 200 <= r.status_code < 300:
+        resp = self._github_api_get(f'repos/jamulussoftware/{self.repo}/commits/{sha}')
+        if 200 <= resp.status_code < 300:
             try:
-                return r.json()['author']['login']
+                return resp.json()['author']['login']
             except (TypeError, KeyError):
-                logger.warning('%s has not github author, saving as empty', hash)
+                logger.warning('%s has not GitHub author, saving as empty', sha)
                 return ''
-        if r.status_code == 422:
-            logger.warning('unable to find author of %s, saving as empty', hash)
+        if resp.status_code == 422:
+            logger.warning('unable to find author of %s, saving as empty', sha)
             return ''
-        raise UnexpectedGithubStatus('status was %d' % r.status_code)
+        raise UnexpectedGithubStatus(f"status was {resp.status_code}")
 
     def _github_api_get(self, path, *args, **kwargs):
         headers = {
             'Accept': 'application/vnd.github.v3+json',
         }
         if self.token:
-            headers['Authorization'] = 'token %s' % self.token
-        r = requests.get(f'https://api.github.com/{path}', *args, headers=headers, **kwargs)
+            headers['Authorization'] = f"token {self.token}"
+        r = requests.get(f'https://api.github.com/{path}', *args, headers=headers, **kwargs,
```

Comment:
timeout=10 was added. Seems sensible. Also why do you use r here but above resp?

Reply:
Fair point, have updated all of these to `resp` which should be clearer